gsa/src/web/components/form/textfield.js

```diff
@@ -18,26 +18,31 @@
 
 import React from 'react';
 
-import Theme from 'web/utils/theme';
+import PropTypes from 'web/utils/proptypes';
 
-import Field from './field.js';
+import Field from './field';
+import {Marker} from './useFormValidation';
 
-const TextField = ({shouldValidate = false, errorContent = '', ...props}) => {
-  const bgColor = shouldValidate ? 'rgb(216, 54, 54, 0.1)' : 'white'; //warningRed at opacity 10%
-  const checkMarkColor = shouldValidate ? Theme.warningRed : 'white';
+const TextField = ({hasError = false, errorContent, title, ...props}) => {
   return (
     <React.Fragment>
       <Field
         {...props}
-        title={errorContent}
-        style={{background: bgColor}}
+        hasError={hasError}
+        title={hasError ? errorContent : title}
```

Comment:
Do we need both title and errorContent? What is the usecase you have in mind?

Reply:
If we have an input field were we would like to actually set a title (as tooltip for example) this would get swallowed here.